hugegraph-server/hugegraph-rocksdb/src/main/java/org/apache/hugegraph/backend/store/rocksdb/RocksDBStore.java

```diff
@@ -813,12 +776,56 @@ private void closeSessions() {
         }
     }
 
-    private Collection<RocksDBSessions> sessions() {
+    private final Collection<RocksDBSessions> sessions() {
         return this.dbs.values();
     }
 
-    private void parseTableDiskMapping(Map<String, String> disks, String dataPath) {
+    private final List<RocksDBSessions.Session> session() {
```

Comment:
Consider making session() method final like sessions() for consistency and to prevent accidental overrides that could break the optimized session management logic.